pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -2006,7 +2016,7 @@ func handleAppNetworkModify(ctxArg interface{}, key string, configArg interface{
 	log.Infof("handleAppNetworkModify(%s) done\n", key)
 	// on resource release, check whether any one else
 	// needs it
-	scanAppNetworkStatusInErrorAndUpdate(ctx, key)
+	scanAppNetworkStatusInErrorAndUpdate(ctx, false)
```

Comment:
I'd define this as a separate function, since it doesn't do an update.
checkAndStartCreateTimer()?

Reply:
will do.